### PR TITLE
add support port

### PR DESCRIPTION
**Pull request by HopscotchSen**
add support for consul port.

### CODE REVIEW

**Comment by mritd** [comment]
@HopscotchSen 
Should we split the address option to host + port?

**Comment by HopscotchSen** [comment]
I think the port should be supported. The default port cannot adapt to certain situations.

File: cclean/cclean.go [hunk 1]
```diff
@@ -26,12 +27,14 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-func Clean(addr, exclude string, timeout time.Duration) {
+func Clean(addr, port int, exclude string, timeout time.Duration) {
```
Comment (mritd) [inline cclean/cclean.go]: Syntax error, compile failed

**Comment by mritd** [comment]
@HopscotchSen Ok, I will split the address later to support the custom port.

**Comment by mritd** [comment]
@HopscotchSen 
[v1.0.3](https://github.com/Gozap/cclean/releases) has been released
custom port feature has been added 😀

**Comment by HopscotchSen** [comment]
Thanks. @mritd

**Pull request merged**